src/logic/documents/scorecards.js

```diff
@@ -329,7 +332,15 @@ const scorecard = ({
   printStations,
   scorecardPaperSize,
   featured = false,
+  language = 'en',
 }) => {
+  const defaultTranslation = translation('en');
+  const translationData = translation(language) || defaultTranslation;
```

Comment:
We should never get a language that is not a valid translation.

```suggestion
  const translationData = translation(language);
```